app/src/main/java/com/portfolio/tasky/ToolbarHandlerImpl.kt

```diff
@@ -0,0 +1,23 @@
+package com.portfolio.tasky
+
+import android.view.animation.TranslateAnimation
+import androidx.appcompat.widget.Toolbar
+import com.portfolio.tasky.globals.Constants
+
+
+class ToolbarHandlerImpl : ToolbarHandler {
+    private lateinit var mToolbar: Toolbar
+    private lateinit var translateAnimation: TranslateAnimation
+
+    override fun setToolBarText(toolbar: Toolbar, text: String) {
+        mToolbar = toolbar
```

Comment:
You are saving a variable "mToolbar" but then are never using it
"mToolbar.title" could just be "toolbar.title" (Toolbar from parameter)

So you could literally remove the variable mToolbar ;)

Reply:
True! 
Done